dsls/pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.beam</groupId>
     <artifactId>beam-parent</artifactId>
-    <version>0.7.0-SNAPSHOT</version>
+    <version>2.1.0-SNAPSHOT</version>
```

Comment:
I would rather not conflate upgrading the underlying version w/ adding join support. Can that be done in a separate PR, please?